### PR TITLE
add Caroussel, no responsive

### PR DESCRIPTION
**Pull request by RomainLF**

### CODE REVIEW

**Review by FrancoisSalmon — DISMISSED** [review]
ok for me

File: frontend/src/components/Caroussel/index.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    setPosition(position + 1);
+  };
+  return (
+    <SCaroussel>
+      <div className="carouselContainer">
+        <button type="button" onClick={prevSlide}>
+          &lsaquo;
+        </button>
+        {players
+          .filter((toto, img) => img >= position && img < position + 4)
```
Comment (LuigiMenez) [inline frontend/src/components/Caroussel/index.jsx:27]: Filter toto seriously??

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Filter TOTO

**Review by FrancoisSalmon — APPROVED** [review]
Saw no more TOTO
\0/

**Pull request merged**